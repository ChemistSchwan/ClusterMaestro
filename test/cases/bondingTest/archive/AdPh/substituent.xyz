96
Properties=species:S:1:pos:R:3:tags:I:1 pbc="F F F"
C      -0.07390949       0.07540704       6.31440102        0 
C       0.84154931      -0.87220168       6.76763993        1 
C      -0.77396106       0.79995011       7.27153648        2 
C       1.04963299      -1.08767535       8.11648606        3 
H       1.40832819      -1.45878812       6.05989858        4 
C      -0.56860108       0.58729784       8.62480276        5 
H      -1.49512014       1.54643349       6.98048154        6 
C       0.34354208      -0.35686541       9.05604877        7 
H       1.76691737      -1.83013385       8.43547359        8 
H      -1.12855853       1.16682550       9.34474580        9 
H       0.50334344      -0.52237087      10.11113123       10 
H      -0.23470768       0.24192214       5.26157815       11 
C      -2.51265052       2.60013815       2.63983727       33 
C      -3.21272725       3.32454362       3.59656913       34 
C      -2.78079450       2.87821383       1.30140568       35 
C      -4.14099357       4.28575959       3.23039767       37 
H      -3.04745657       3.15303352       4.64796978       40 
C      -3.70493150       3.83517947       0.93019480       36 
H      -2.25934473       2.33854807       0.52569863       39 
C      -4.39308649       4.54731621       1.89707401       38 
H      -4.66976182       4.83291784       3.99759820       42 
H      -3.88861411       4.02575626      -0.11752486       41 
H      -5.11674181       5.29673349       1.61289571       43 
H      -1.79037335       1.85230324       2.92457916        0 
C       2.25204932       1.31832939       2.07555921       44 
C       2.97960546       0.54184138       1.18274801       45 
C       2.77151157       2.56739436       2.40689193       46 
C       4.15403328       1.00332196       0.61336634       48 
H       2.64824653      -0.44952188       0.92059051       51 
C       3.94582095       3.03157370       1.84635852       47 
H       2.25717502       3.19695239       3.11749531       50 
C       4.63973027       2.25551842       0.93504563       49 
H       4.68764329       0.37466829      -0.08507702       53 
H       4.32004148       4.00688234       2.12185931       52 
H       5.55228678       2.62038213       0.48812292       54 
H       1.32302234       0.96626020       2.49385990        0 
C      -1.39580586      -2.20600078       2.07523477       55 
C      -0.64461137      -2.95989117       1.18262245       56 
C      -2.66208531      -2.68215594       2.40628729       57 
C      -1.14610745      -4.11776870       0.61317457       59 
H       0.35761353      -2.66285546       0.92062315       62 
C      -3.16628293      -3.83980898       1.84567419       58 
H      -3.27375959      -2.14640744       3.11669091       61 
C      -2.41435765      -4.56006362       0.93457554       60 
H      -0.53604333      -4.67269595      -0.08514458       64 
H      -4.15398542      -4.18019590       2.12090805       63 
H      -2.81031624      -5.45952957       0.48757719       65 
H      -1.01207122      -1.28962555       2.49359285        0 
C       0.07391349      -0.07583970      -6.31446158        0 
C      -0.84167819       0.87156756      -6.76785410        1 
C       0.77402076      -0.80048424      -7.27147926        2 
C      -1.04985209       1.08673631      -8.11673472        3 
H      -1.40850140       1.45822613      -6.06020849        4 
C       0.56857108      -0.58813672      -8.62478016        5 
H       1.49528882      -1.54681467      -6.98030178        6 
C      -0.34371842       0.35581543      -9.05617886        7 
H      -1.76724685       1.82903709      -8.43584099        8 
H       1.12856468      -1.16774537      -9.34462981        9 
H      -0.50360236       0.52107117     -10.11128796       10 
H       0.23476932      -0.24212868      -5.26161178       11 
C       2.51297497      -2.59964017      -2.63944552       33 
C       3.21344085      -3.32386090      -3.59603040       34 
C       2.78098955      -2.87756971      -1.30095775       35 
C       4.14197518      -4.28474356      -3.22966072       37 
H       3.04827367      -3.15245978      -4.64746546       40 
C       3.70539530      -3.83419727      -0.92954878       36 
H       2.25922211      -2.33804352      -0.52536873       39 
C       4.39395397      -4.54613980      -1.89628383       38 
H       4.67105392      -4.83176053      -3.99674776       42 
H       3.88897945      -4.02465574       0.11821009       41 
H       5.11783379      -5.29528216      -1.61195241       43 
H       1.79050314      -1.85205118      -2.92433933        0 
C      -2.25201010      -1.31861961      -2.07564403       44 
C      -2.97958522      -0.54238701      -1.18263139       45 
C      -2.77142429      -2.56761855      -2.40730078       46 
C      -4.15402061      -1.00404205      -0.61340662       48 
H      -2.64823068       0.44890464      -0.92019639       51 
C      -3.94575091      -3.03195931      -1.84694315       47 
H      -2.25702778      -3.19699348      -3.11802453       50 
C      -4.63970288      -2.25614958      -0.93545416       49 
H      -4.68765171      -0.37560180       0.08520960       53 
H      -4.31995182      -4.00719814      -2.12271977       52 
H      -5.55228461      -2.62113205      -0.48868103       54 
H      -1.32299788      -0.96640671      -2.49385651        0 
C       1.39537438       2.20627975      -2.07547926       55 
C       0.64427589       2.95996502      -1.18261744       56 
C       2.66148323       2.68268431      -2.40682683       57 
C       1.14571783       4.11788537      -0.61320729       59 
H      -0.35782726       2.66271857      -0.92039251       62 
C       3.16561940       3.84038631      -1.84626393       58 
H       3.27307354       2.14708202      -3.11741407       61 
C       2.41380624       4.56042964      -0.93490510       60 
H       0.53574239       4.67265037       0.08531761       64 
H       4.15318960       4.18097059      -2.12173009       63 
H       2.80972278       5.45993206      -0.48794236       65 
H       1.01169356       1.28986736      -2.49380530        0 
