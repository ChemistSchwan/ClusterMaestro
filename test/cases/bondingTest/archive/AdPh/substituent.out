      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. MÃ¤rz, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/09/25 at 10:01:15.667     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : xtb start.xyz -P 1
          coordinate file            : start.xyz
          omp threads                :                     1
          number of atoms            :                    96
          number of electrons        :                   240
          charge                     :                     0
          spin                       :                   0.0
          first test random number   :      0.51116087727653


molecular fragmentation (1/2 indicates fragments):
111111111111222222222222222222222222222222222222222222222222222222222222
222222222222222222222222
# atoms in fragment 1/2:    12    84
 fragment masses (1/2) :       78.11      546.78
CMA distance (Bohr)    :  16.619
constraining FC (au)   :  0.0500
########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                 240          :
          :  # atomic orbitals                 240          :
          :  # shells                          144          :
          :  # electrons                       240          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1   -129.4696276 -0.129470E+03  0.394E+00    4.80       0.0  T
   2   -129.5833596 -0.113732E+00  0.230E+00    4.78       1.0  T
   3   -129.5873050 -0.394543E-02  0.513E-01    4.72       1.0  T
   4   -129.5890012 -0.169624E-02  0.534E-02    4.70       1.0  T
   5   -129.5890088 -0.751307E-05  0.323E-02    4.70       1.0  T
   6   -129.5890522 -0.434260E-04  0.478E-03    4.71       2.1  T
   7   -129.5890539 -0.175135E-05  0.204E-03    4.71       5.0  T
   8   -129.5890544 -0.445438E-06  0.563E-04    4.71      18.1  T
   9   -129.5890544 -0.587343E-08  0.284E-04    4.71      36.0  T

   *** convergence criteria satisfied after 9 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6431411             -17.5008
       ...           ...                  ...                  ...
       114        2.0000           -0.4021396             -10.9428
       115        2.0000           -0.4020577             -10.9405
       116        2.0000           -0.4020542             -10.9405
       117        2.0000           -0.3976873             -10.8216
       118        2.0000           -0.3976823             -10.8215
       119        2.0000           -0.3974906             -10.8163
       120        2.0000           -0.3974872             -10.8162 (HOMO)
       121                         -0.2245516              -6.1104 (LUMO)
       122                         -0.2245507              -6.1103
       123                         -0.2229463              -6.0667
       124                         -0.2229462              -6.0667
       125                         -0.2223837              -6.0514
       ...                                ...                  ...
       240                          0.6801631              18.5082
      -------------------------------------------------------------
                  HL-Gap            0.1729356 Eh            4.7058 eV
             Fermi-level           -0.3110194 Eh           -8.4633 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.399 sec
 SCC setup                      ...        0 min,  0.002 sec (  0.470%)
 Dispersion                     ...        0 min,  0.005 sec (  1.134%)
 integral evaluation            ...        0 min,  0.127 sec ( 31.883%)
 zeroth order Hamiltonian       ...        0 min,  0.001 sec (  0.215%)
 iterations                     ...        0 min,  0.162 sec ( 40.679%)
 molecular gradient             ...        0 min,  0.099 sec ( 24.881%)
 printout                       ...        0 min,  0.003 sec (  0.726%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy            -127.055282302976 Eh    ::
         :: gradient norm              0.053003986724 Eh/a0 ::
         :: HOMO-LUMO gap              4.705818175340 eV    ::
         ::.................................................::
         :: SCC energy              -129.589054382130 Eh    ::
         :: -> isotropic ES            0.006134168351 Eh    ::
         :: -> anisotropic ES          0.023774844669 Eh    ::
         :: -> anisotropic XC          0.098602902609 Eh    ::
         :: -> dispersion             -0.093260605650 Eh    ::
         :: repulsion energy           2.531752455547 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6431411             -17.5008
       ...           ...                  ...                  ...
       108        2.0000           -0.4039342             -10.9916
       109        2.0000           -0.4030575             -10.9678
       110        2.0000           -0.4030515             -10.9676
       111        2.0000           -0.4028224             -10.9614
       112        2.0000           -0.4028221             -10.9613
       113        2.0000           -0.4021397             -10.9428
       114        2.0000           -0.4021396             -10.9428
       115        2.0000           -0.4020577             -10.9405
       116        2.0000           -0.4020542             -10.9405
       117        2.0000           -0.3976873             -10.8216
       118        2.0000           -0.3976823             -10.8215
       119        2.0000           -0.3974906             -10.8163
       120        2.0000           -0.3974872             -10.8162 (HOMO)
       121                         -0.2245516              -6.1104 (LUMO)
       122                         -0.2245507              -6.1103
       123                         -0.2229463              -6.0667
       124                         -0.2229462              -6.0667
       125                         -0.2223837              -6.0514
       126                         -0.2223809              -6.0513
       127                         -0.2222214              -6.0470
       128                         -0.2222193              -6.0469
       129                         -0.2211539              -6.0179
       130                         -0.2211496              -6.0178
       131                         -0.2205445              -6.0013
       ...                                ...                  ...
       240                          0.6801631              18.5082
      -------------------------------------------------------------
                  HL-Gap            0.1729356 Eh            4.7058 eV
             Fermi-level           -0.3110194 Eh           -8.4633 eV

     #   Z        covCN         q      C6AA      Î±(0)
     1   6 C      2.912    -0.028    28.698     8.784
     2   6 C      2.925    -0.028    28.699     8.785
     3   6 C      2.924    -0.028    28.710     8.787
     4   6 C      2.919    -0.029    28.715     8.787
     5   1 H      0.926     0.028     2.611     2.527
     6   6 C      2.920    -0.029    28.735     8.790
     7   1 H      0.926     0.027     2.629     2.536
     8   6 C      2.919    -0.028    28.710     8.787
     9   1 H      0.926     0.029     2.601     2.522
    10   1 H      0.926     0.028     2.609     2.526
    11   1 H      0.926     0.029     2.603     2.523
    12   1 H      0.926     0.029     2.598     2.521
    13   6 C      2.912    -0.028    28.693     8.784
    14   6 C      2.924    -0.028    28.707     8.786
    15   6 C      2.926    -0.026    28.651     8.778
    16   6 C      2.920    -0.029    28.728     8.789
    17   1 H      0.926     0.027     2.627     2.534
    18   6 C      2.919    -0.029    28.714     8.787
    19   1 H      0.926     0.025     2.657     2.549
    20   6 C      2.920    -0.028    28.709     8.787
    21   1 H      0.926     0.028     2.608     2.525
    22   1 H      0.926     0.029     2.600     2.521
    23   1 H      0.926     0.029     2.603     2.523
    24   1 H      0.926     0.030     2.590     2.516
    25   6 C      2.912    -0.028    28.702     8.785
    26   6 C      2.926    -0.028    28.689     8.784
    27   6 C      2.926    -0.028    28.701     8.786
    28   6 C      2.920    -0.029    28.714     8.787
    29   1 H      0.926     0.026     2.641     2.542
    30   6 C      2.920    -0.028    28.704     8.786
    31   1 H      0.926     0.028     2.620     2.531
    32   6 C      2.920    -0.027    28.687     8.783
    33   1 H      0.926     0.029     2.606     2.525
    34   1 H      0.926     0.028     2.611     2.527
    35   1 H      0.926     0.028     2.613     2.528
    36   1 H      0.926     0.029     2.603     2.523
    37   6 C      2.912    -0.028    28.702     8.785
    38   6 C      2.926    -0.028    28.689     8.784
    39   6 C      2.926    -0.028    28.701     8.786
    40   6 C      2.920    -0.029    28.714     8.787
    41   1 H      0.926     0.026     2.642     2.542
    42   6 C      2.920    -0.028    28.704     8.786
    43   1 H      0.926     0.028     2.620     2.531
    44   6 C      2.920    -0.027    28.687     8.783
    45   1 H      0.926     0.029     2.606     2.525
    46   1 H      0.926     0.028     2.611     2.527
    47   1 H      0.926     0.028     2.613     2.528
    48   1 H      0.926     0.029     2.603     2.523
    49   6 C      2.912    -0.028    28.698     8.784
    50   6 C      2.925    -0.028    28.699     8.785
    51   6 C      2.924    -0.028    28.710     8.787
    52   6 C      2.919    -0.029    28.715     8.787
    53   1 H      0.926     0.028     2.611     2.527
    54   6 C      2.920    -0.029    28.735     8.790
    55   1 H      0.926     0.027     2.629     2.536
    56   6 C      2.919    -0.028    28.710     8.787
    57   1 H      0.926     0.029     2.601     2.522
    58   1 H      0.926     0.028     2.609     2.526
    59   1 H      0.926     0.029     2.603     2.523
    60   1 H      0.926     0.029     2.598     2.521
    61   6 C      2.912    -0.028    28.693     8.784
    62   6 C      2.924    -0.028    28.707     8.786
    63   6 C      2.926    -0.026    28.651     8.778
    64   6 C      2.920    -0.029    28.728     8.789
    65   1 H      0.926     0.027     2.627     2.534
    66   6 C      2.919    -0.029    28.714     8.787
    67   1 H      0.926     0.025     2.657     2.549
    68   6 C      2.920    -0.028    28.709     8.787
    69   1 H      0.926     0.028     2.608     2.525
    70   1 H      0.926     0.029     2.600     2.521
    71   1 H      0.926     0.029     2.603     2.523
    72   1 H      0.926     0.030     2.590     2.516
    73   6 C      2.912    -0.028    28.702     8.785
    74   6 C      2.926    -0.028    28.689     8.784
    75   6 C      2.926    -0.028    28.701     8.786
    76   6 C      2.920    -0.029    28.714     8.787
    77   1 H      0.926     0.026     2.642     2.542
    78   6 C      2.920    -0.028    28.704     8.786
    79   1 H      0.926     0.028     2.620     2.531
    80   6 C      2.920    -0.027    28.687     8.783
    81   1 H      0.926     0.029     2.606     2.525
    82   1 H      0.926     0.028     2.611     2.527
    83   1 H      0.926     0.028     2.613     2.528
    84   1 H      0.926     0.029     2.603     2.523
    85   6 C      2.912    -0.028    28.702     8.785
    86   6 C      2.926    -0.028    28.689     8.784
    87   6 C      2.926    -0.028    28.701     8.786
    88   6 C      2.920    -0.029    28.714     8.787
    89   1 H      0.926     0.026     2.642     2.542
    90   6 C      2.920    -0.028    28.704     8.786
    91   1 H      0.926     0.028     2.620     2.531
    92   6 C      2.920    -0.027    28.687     8.783
    93   1 H      0.926     0.029     2.606     2.525
    94   1 H      0.926     0.028     2.611     2.527
    95   1 H      0.926     0.028     2.613     2.528
    96   1 H      0.926     0.029     2.603     2.523

 Mol. C6AA /auÂ·bohrâ¶  :     112010.114881
 Mol. C8AA /auÂ·bohrâ¸  :    2730670.295615
 Mol. Î±(0) /au        :        543.056240


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  C   3.992        C    3 1.444    C    2 1.434    H   12 0.972    C    8 0.116
     2  C   3.993        C    4 1.449    C    1 1.434    H    5 0.972    C    6 0.113
     3  C   3.993        C    1 1.444    C    6 1.439    H    7 0.972    C    4 0.112
     4  C   3.993        C    2 1.449    C    8 1.436    H    9 0.971    C    3 0.112
     5  H   0.999        C    2 0.972
     6  C   3.993        C    8 1.445    C    3 1.439    H   10 0.972    C    2 0.113
     7  H   0.999        C    3 0.972
     8  C   3.993        C    6 1.445    C    4 1.436    H   11 0.971    C    1 0.116
     9  H   0.999        C    4 0.971
    10  H   0.999        C    6 0.972
    11  H   0.999        C    8 0.971
    12  H   0.999        C    1 0.972
    13  C   3.992        C   14 1.445    C   15 1.433    H   24 0.972    C   20 0.116
    14  C   3.993        C   13 1.445    C   16 1.438    H   17 0.972    C   18 0.112
    15  C   3.993        C   18 1.449    C   13 1.433    H   19 0.967    C   16 0.113
    16  C   3.993        C   20 1.446    C   14 1.438    H   21 0.972    C   15 0.113
    17  H   0.999        C   14 0.972
    18  C   3.993        C   15 1.449    C   20 1.435    H   22 0.971    C   14 0.112
    19  H   0.999        C   15 0.967
    20  C   3.993        C   16 1.446    C   18 1.435    H   23 0.971    C   13 0.116
    21  H   0.999        C   16 0.972
    22  H   0.999        C   18 0.971
    23  H   0.999        C   20 0.971
    24  H   0.999        C   13 0.972
    25  C   3.992        C   26 1.444    C   27 1.434    H   36 0.972    C   32 0.116
    26  C   3.993        C   25 1.444    C   28 1.439    H   29 0.969    C   30 0.112
    27  C   3.993        C   30 1.448    C   25 1.434    H   31 0.972    C   28 0.113
    28  C   3.993        C   32 1.445    C   26 1.439    H   33 0.971    C   27 0.113
    29  H   0.999        C   26 0.969
    30  C   3.993        C   27 1.448    C   32 1.436    H   34 0.971    C   26 0.112
    31  H   0.999        C   27 0.972
    32  C   3.993        C   28 1.445    C   30 1.436    H   35 0.971    C   25 0.116
    33  H   0.999        C   28 0.971
    34  H   0.999        C   30 0.971
    35  H   0.999        C   32 0.971
    36  H   0.999        C   25 0.972
    37  C   3.992        C   38 1.444    C   39 1.434    H   48 0.972    C   44 0.116
    38  C   3.993        C   37 1.444    C   40 1.439    H   41 0.969    C   42 0.112
    39  C   3.993        C   42 1.448    C   37 1.434    H   43 0.972    C   40 0.113
    40  C   3.993        C   44 1.445    C   38 1.439    H   45 0.971    C   39 0.113
    41  H   0.999        C   38 0.969
    42  C   3.993        C   39 1.448    C   44 1.436    H   46 0.971    C   38 0.112
    43  H   0.999        C   39 0.972
    44  C   3.993        C   40 1.445    C   42 1.436    H   47 0.971    C   37 0.116
    45  H   0.999        C   40 0.971
    46  H   0.999        C   42 0.971
    47  H   0.999        C   44 0.971
    48  H   0.999        C   37 0.972
    49  C   3.992        C   51 1.444    C   50 1.434    H   60 0.972    C   56 0.116
    50  C   3.993        C   52 1.449    C   49 1.434    H   53 0.972    C   54 0.113
    51  C   3.993        C   49 1.444    C   54 1.439    H   55 0.972    C   52 0.112
    52  C   3.993        C   50 1.449    C   56 1.436    H   57 0.971    C   51 0.112
    53  H   0.999        C   50 0.972
    54  C   3.993        C   56 1.445    C   51 1.439    H   58 0.972    C   50 0.113
    55  H   0.999        C   51 0.972
    56  C   3.993        C   54 1.445    C   52 1.436    H   59 0.971    C   49 0.116
    57  H   0.999        C   52 0.971
    58  H   0.999        C   54 0.972
    59  H   0.999        C   56 0.971
    60  H   0.999        C   49 0.972
    61  C   3.992        C   62 1.445    C   63 1.433    H   72 0.972    C   68 0.116
    62  C   3.993        C   61 1.445    C   64 1.438    H   65 0.972    C   66 0.112
    63  C   3.993        C   66 1.449    C   61 1.433    H   67 0.967    C   64 0.113
    64  C   3.993        C   68 1.446    C   62 1.438    H   69 0.972    C   63 0.113
    65  H   0.999        C   62 0.972
    66  C   3.993        C   63 1.449    C   68 1.435    H   70 0.971    C   62 0.112
    67  H   0.999        C   63 0.967
    68  C   3.993        C   64 1.446    C   66 1.435    H   71 0.971    C   61 0.116
    69  H   0.999        C   64 0.972
    70  H   0.999        C   66 0.971
    71  H   0.999        C   68 0.971
    72  H   0.999        C   61 0.972
    73  C   3.992        C   74 1.444    C   75 1.434    H   84 0.972    C   80 0.116
    74  C   3.993        C   73 1.444    C   76 1.439    H   77 0.969    C   78 0.112
    75  C   3.993        C   78 1.448    C   73 1.434    H   79 0.972    C   76 0.113
    76  C   3.993        C   80 1.445    C   74 1.439    H   81 0.971    C   75 0.113
    77  H   0.999        C   74 0.969
    78  C   3.993        C   75 1.448    C   80 1.436    H   82 0.971    C   74 0.112
    79  H   0.999        C   75 0.972
    80  C   3.993        C   76 1.445    C   78 1.436    H   83 0.971    C   73 0.116
    81  H   0.999        C   76 0.971
    82  H   0.999        C   78 0.971
    83  H   0.999        C   80 0.971
    84  H   0.999        C   73 0.972
    85  C   3.992        C   86 1.444    C   87 1.434    H   96 0.972    C   92 0.116
    86  C   3.993        C   85 1.444    C   88 1.439    H   89 0.969    C   90 0.112
    87  C   3.993        C   90 1.448    C   85 1.434    H   91 0.972    C   88 0.113
    88  C   3.993        C   92 1.445    C   86 1.439    H   93 0.971    C   87 0.113
    89  H   0.999        C   86 0.969
    90  C   3.993        C   87 1.448    C   92 1.436    H   94 0.971    C   86 0.112
    91  H   0.999        C   87 0.972
    92  C   3.993        C   88 1.445    C   90 1.436    H   95 0.971    C   85 0.116
    93  H   0.999        C   88 0.971
    94  H   0.999        C   90 0.971
    95  H   0.999        C   92 0.971
    96  H   0.999        C   85 0.972

molecular dipole:
                 x           y           z       tot (Debye)
 q only:       -0.000      -0.000       0.000
   full:       -0.000      -0.000      -0.000       0.000
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:       -0.308      -2.173      -0.159      -2.236       2.314       0.468
  q+dip:       -1.602      -9.169      -0.974     -10.548      10.914       2.576
   full:       -0.657      -4.520      -0.348      -5.196       5.377       1.005


           -------------------------------------------------
          | TOTAL ENERGY             -127.055282302976 Eh   |
          | GRADIENT NORM               0.053003986724 Eh/Î± |
          | HOMO-LUMO GAP               4.705818175340 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/09/25 at 10:01:16.190     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.523 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.522 sec
 * ratio c/w:     0.999 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.400 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.399 sec
 * ratio c/w:     0.999 speedup

